### PR TITLE
Adden hidden input and name attribute 

### PR DESCRIPTION
**Pull request by hankphung**

### CODE REVIEW

File: paper-autocomplete.html [hunk 1]
```diff
@@ -128,7 +128,7 @@
     <div class="input-wrapper" role="combobox" aria-haspopup="true" aria-owns="suggestionsWrapper" aria-expanded$="[[_isSuggestionsOpened]]">
       <!-- For accessibility, it is needed to have a label or aria-label. Label is preferred -->
       <label for="autocompleteInput" class="sr-only">[[label]]</label>
-
+      <paper-input   name$="[[name]]" value="{{_selectedOption.value}}" hidden ></paper-input >
```
Comment (jhuesos) [inline paper-autocomplete.html:131]: can you explain what did you try to do here? I am not sure I follow. What is the purpose of this change?

File: paper-autocomplete.html [hunk 2]
```diff
@@ -144,6 +144,7 @@
                    always-float-label="[[alwaysFloatLabel]]"
                    char-counter$="[[charCounter]]"
                    maxlength$="[[maxlength]]"
+
```
Comment (jhuesos) [inline paper-autocomplete.html:147]: don't add additional extra lines if not needed.

**Review by jhuesos — COMMENTED** [review]
I am not sure what did you try to do here. Why do we need an additional paper-input there.

If what you are trying to do here is to have a hidden input to send the value back to the server and connect it to an iron-form, as far as I know you don't need a paper-input, just a normal input with a name field. Or am I mistaken?

By the way, thanks a lot for the contribution.

EDIT: sorry I forgot to mention a few things missing, there are no test for this change.

**Comment by jhuesos** [comment]
I hope you don't mind, but I started doing this change myself. If you want to help, maybe you can test it with your iron-form. I am closing this PR.

To test, install bower using the version from branch `feature/iron-form-integration` and tell me how did go. I am going to do my best to finish it tomorrow, but if you can test that works, that would help me to speed up.

Thanks!

**Comment by jhuesos** [comment]
This is PR I open (Work in Progress) with the change I mentioned before:

https://github.com/ellipticaljs/paper-autocomplete/pull/46

File: paper-autocomplete.html (hunk 1 above)
Comment (hankphung) [reply paper-autocomplete.html:131]: This hidden input will represent the given name on paper-autocomplete, It will get the selected value of the component. On form submit, this will make the selected value present in form data.
It just my work around, never mind.

**Comment by jhuesos** [comment]
@dannyphung Added functionality in version `v2.2.0`. Thanks for the feedback

**Comment by hankphung** [comment]
great!!! thank you @jhuesos
